x-pack/solutions/security/packages/kbn-cloud-security-posture/graph/src/components/node/pentagon_node.tsx

```diff
@@ -55,6 +67,7 @@ export const PentagonNode = memo<NodeProps>((props: NodeProps) => {
           viewBox="0 0 75 72"
           fill="none"
           xmlns="http://www.w3.org/2000/svg"
+          isActive={dragging}
```

Comment:
In case we apply this `active` designs, I think we should make sure it appears not only on dragging. 
We need to define active states. IMO it won't make sense to do all this effort if it only applied while dragging.